sequelizeDatabase/modelSetup.js

```diff
@@ -0,0 +1,135 @@
+const Sequelize = require('sequelize');
+
+const sequelize = new Sequelize('localDatabaseSean','root','ConnectedNetwork1',{
+    host: 'localhost',
+    dialect: 'mysql',
+    define: {
+      freezeTableName: true
+    }
+  });
+
+sequelize.authenticate()
+  .then(() => {
+    console.log('Connection has been established successfully.');
+  })
+  .catch(err => {
+    console.error('Unable to connect to the database:', err);
+  });
+
+  const Individuals = sequelize.define('individuals',{
```

Comment:
From here on you added extra indentation. Happens, might be able to look for something in your IDE to notify you of that.